{"name": "账号或密码长度不够", "status": "broken", "statusDetails": {"message": "AttributeError: 'TestLogin' object has no attribute 'data'", "trace": "self = <Testcase.test_one.TestLogin object at 0x000002324EE4AA00>\n\n    @allure.title(\"账号或密码长度不够\")\n    def test_02(self):\n>       user, pwd, hope = self.data[2].values()\nE       AttributeError: 'TestLogin' object has no attribute 'data'\n\nTestcase\\test_one.py:36: AttributeError"}, "start": 1628646183460, "stop": 1628646183460, "uuid": "400c30be-38d5-4fdf-a797-6a36fb51b1d7", "historyId": "e15b800d7b262cf98239a48491a42620", "testCaseId": "fae3abaef285e96b00acd5e302a2bb92", "fullName": "Testcase.test_one.TestLogin#test_02", "labels": [{"name": "feature", "value": "编辑页面"}, {"name": "parentSuite", "value": "Testcase"}, {"name": "suite", "value": "test_one"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "SK-20210625NVCA"}, {"name": "thread", "value": "7132-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Testcase.test_one"}]}